### DISCUSSION TITLE
Request: Add Legacy Computing Symbols

### DISCUSSION THREAD

**Discussion by USLTD** [ryanoasis/nerd-fonts#835] [Ideas]
https://unicode.org/versions/Unicode13.0.0/

**Comment by ChiefMikeK** [comment]
Should Already be in Iosevka  
https://github.com/ryanoasis/nerd-fonts/tree/master/patched-fonts/Iosevka

**Reply by Finii** [reply]
Mind that a lot of patched Iosevkas are broken because `fontforge` before `March 2022 Release` shredders some tables.

#806 pulls that fixed release at least in, but iirc the 2.2.0 RC and all other 'normal' CI patching is done with an older `fontforge` version :-/

**Reply by USLTD** [reply]
Are there any in FiraCode?

**Reply by Finii** [reply]
I guess the Request is about these: https://en.wikipedia.org/wiki/Symbols_for_Legacy_Computing

Codepoints `1FB00` - `1FB92`, `1FB94` - `1FBCA`, `1FBF0` - `1FBF9` if the Wikipedia page has it correct.

Nerd Fonts does not patch them in anywhere. If some font (in your case FiraCode) has them originally they will of course be preserved.

Quick check with `fontforge` ... the unpatched FiraCode in this repo does not have any of them:

![image](https://user-images.githubusercontent.com/16012374/170018851-ceeada03-0de7-4366-9a8d-19dd0dcecff7.png)

(Would be listed after the last entry, with `U+1FB00` ...)

**Reply by Finii** [reply]
Counter example, Iosevka, as ChiefMikeK pointed out:

![image](https://user-images.githubusercontent.com/16012374/170019778-6a2cda51-f1eb-4f20-82c8-6dbff48d3be1.png)

**Reply by USLTD** [reply]
Alright.

**Comment by ChiefMikeK** [comment]
![copyPasteTest](https://user-images.githubusercontent.com/20070053/170024590-f37233f0-2194-4ae6-bd20-6ff2ee595e3c.png)
🯀🯁🯂🯃🯄🯅🯆🯇🯈🯉🯊🯰🯱🯲🯳🯴🯵🯶🯷🯸🯹🮲🮳🢰🢱

**Reply by ChiefMikeK** [reply]
You could edit your `/etc/fonts.conf` fallback-font to Iosveka

**Reply by Finii** [reply]
There is no need to edit the fonts conf, nowadays.

I just installed iosevka, locally, and - without any font-cache-reload stuff that some people still claim is needed, I get

![image](https://user-images.githubusercontent.com/16012374/170033134-8510e83d-e9ca-4cb5-9e38-da6cbfc8acf7.png)

On all systems I work with that is done automagically, and - apart from installing the font - no need to do anything to get the font fallback (if it is supported by your terminal).
